### PR TITLE
TGBOT-49 Add disable loading data flag

### PR DESCRIPTION
**Pull request by schneidervs**
https://schneidervs.atlassian.net/browse/TGBOT-49

### CODE REVIEW

**Review by AlexanderYarokhovich — APPROVED** [review]

**Comment by schneidervs** [comment]
-edl == EnableLoaderData
-dld == DisableLoaderData
By default, the application tends not to load anything.

**Review by Sav4ykVladzislau — APPROVED** [review]

File: stoservice/src/main/java/com/tutrit/stoservice/settings/ArgsSetup.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    Configurations configurations;
+
+    public ArgsSetup(Configurations configurations) {
+        this.configurations = configurations;
+    }
+
+    public void readingFlags(String[] args) {
+        if (args != null) for (String arg : args) {
+            if (arg.equals("-eld")) configurations.setLoadDataDemo(true);
+            if (arg.equals("-dld")) configurations.setLoadDataDemo(false);
```
Comment (yurasava) [inline stoservice/src/main/java/com/tutrit/stoservice/settings/ArgsSetup.java:13]: What "eld" and "dld" mean?

File: stoservice/src/main/java/com/tutrit/stoservice/StoserviceApplication.java [hunk 2]
```diff
@@ -16,6 +17,7 @@ public class StoserviceApplication {
     public static void main(String[] args) {
         SpringApplication.run(StoserviceApplication.class, args);
         ApplicationContextLoader.run();
+        if (args.length > 0) get(ArgsSetup.class).readingFlags(args);
```
Comment (yurasava) [inline stoservice/src/main/java/com/tutrit/stoservice/StoserviceApplication.java:20]: The "if" condition must be moved inside the "readingFlags()" method.
You will get an NullPointerException if the args is null

Comment (schneidervs) [reply stoservice/src/main/java/com/tutrit/stoservice/StoserviceApplication.java:20]: The main method does not create null, it creates an expression (with an empty argument)
new String[0].
foreach will take an empty value without any problem.
```
String[] str = new String[0];
        System.out.println(str.length);
        for (String a:str) {
            System.out.println(a);
        }
```
This construct does not create exceptions

**Review by yurasava — APPROVED** [review]

**Review by Sav4ykVladzislau — APPROVED** [review]

**Review by AlexanderYarokhovich — APPROVED** [review]

**Pull request merged**